website/profile/views.py

```diff
@@ -371,7 +371,7 @@ def user_addons(auth, **kwargs):
 def user_notifications(auth, **kwargs):
```

Comment:
Can you rename this to `get_user_mailing_lists`? (or something similar) Now that we have a notifications system, this name might be confusing.